README.md

```diff
@@ -1,2 +1,172 @@
-# budget-mate
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>BudgetMate</b></h3>
+  <img align="center" src="https://user-images.githubusercontent.com/111214189/231619850-f45b1c75-c757-4be6-b224-18a0d4ffa388.png" />
+
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Table of Contents](#-table-of-contents)
+- [📖 My Blog App ](#-my-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BudgetMate <a name="about-project"></a>
+
 BudgetMate it's a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add category
+- Show categories
+- Add transaction
+- Show transactions for each category
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] Great job on the design but I think it would be great if you deploy the project to be  accessible online as required in the requirements so kindly work on the deployement and don't forget to add it in your README file 🚀🤩💪🏻

![image](https://user-images.githubusercontent.com/58553711/231633085-e7ab91e7-af5a-41ab-b732-5e89a18b2e23.png)